src/routes/+layout.svelte

```diff
@@ -64,6 +71,16 @@
 </script>
 
 <svelte:head>
+  <script>
+    // https://partytown.builder.io/sveltekit
```

Comment:
Highly recommend giving this a look as well as the other docs to get a better understanding of what's going on here.

* [Primer](https://partytown.builder.io/how-does-partytown-work)
* [How forwarding works](https://partytown.builder.io/forwarding-events)
* [SvelteKit Tutorial](https://partytown.builder.io/sveltekit)

Some differences between the tutorial and what I've done here are:
* We also forward `gtag` so that we can call that instead of `dataLayer.push`, since I think that `gtag` is more clear (many things can use `dataLayer`, but `gtag` is specific to GA). We edit the boilerplate GA setup script so that we declare `gtag` on the `window` so we can access it in the main thread.
* We're currently not reverse-proxying any scripts. It seems like the latest version of `partytown` handles this automatically. If we notice CORS errors once deployed to Vercel, we can use [this older version of the SvelteKit tutorial](https://monogram.io/blog/add-partytown-to-svelte) as a rough guide to setting up rewrites in Vercel to handle reverse-proxying.
* We're doing a little extra work to ensure we're not hard-coding the GA ID (instead using an env var), which makes the whole implementation a little less easy-to-read but more maintainable.